Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/download/DefaultAppConfigSource.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.appconfig.download
+
+import android.content.Context
+import dagger.Reusable
+import de.rki.coronawarnapp.util.HashExtensions.toSHA256
+import de.rki.coronawarnapp.util.di.AppContext
+import javax.inject.Inject
+
+@Reusable
+class DefaultAppConfigSource @Inject constructor(
```

Comment:
For now, this should only be required in the tests, so I would suggest to move it there.

Reply:
Why would this only be required in tests?

Reply:
The SHA256 thingy is only required for tests